tests/test_bake_project.py

```diff
@@ -43,33 +45,52 @@ def inside_dir(dirpath):
         os.chdir(old_path)
 
 
+def assert_command(command):
+    """If a command exits with a status code different from 0, the output will be printed"""
```

Comment:
I like this one, you should add it as well in pytest-simcore testers
BTW: we started to add types everywhere, specially with the new functionality. Here it would be
```python

def assert_command(command: str) -> None:
    ... 
```

Reply:
added the type and also added a note to copy this command